client/components/infinite-list/scroll-store.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import debugFactory from 'debug';
+import { throttle } from 'lodash';
+
+const debug = debugFactory( 'calypso:infinite-list:scroll-store' );
+
+const THROTTLE_INTERVAL_MS = 1000;
+const THROTTLE_OPTIONS = { leading: false };
```

Comment:
double checking: this means it takes the latest call right instead of the first call right?

Reply:
As I understand `throttle`, the default behavior with `leading: true` is to call the throttled function on the very first scroll event and then call it no earlier than after 1000ms. With `leading: false`, the first call happens 1000ms after the first scroll event.

But it doesn't really matter, as you made a good point that the throttling is not needed at all if there are no change events and listeners.

Reply:
Hmm, now when I'm testing the un-throttled `store` methods, it breaks the scroll position storing functionality. The `scrollTop` stored value is reset to 0 when navigating away from a route like `/media`. The `InfiniteList` scrolling code seems to be not 100% correct. But I don't want to go down that rabbit hole now: let's just de-Fluxify the stores.